client/src/constants/backend_constants.js

```diff
@@ -1,11 +1,89 @@
-import deepFreeze from 'deep-freeze';
 import generatedBackendConstants from '@/../generated/backend_constants.json';
+import forms from '@/constants/forms.js';
+import deepFreeze from 'deep-freeze';
 
 /**
  * Note that these are generated by running `npm run setup` first.
  *
  * @typedef HMModel
  */
-const BACKEND_CONSTANTS = deepFreeze(generatedBackendConstants);
+const mapping = {
+  growth: '_GrowthFactor_params',
+  mdef: 'MassDefinition_params',
+  cosmo: 'cosmo_defaults',
+  filter: 'Filter_params',
+  hmf: 'FittingFunction_params',
+  hod: 'HOD_params',
+  bias: 'Bias_params',
+  halo_concentration: 'CMRelation_params',
+  tracer_concentration: 'CMRelation_params',
+  halo_profile: 'Profile_params',
+  tracer_profile: 'Profile_params',
+  halo_exclusion: 'Exclusion_params',
+  transfer: 'TransferComponent_params',
+};
```

Comment:
Could these be put in the `forms.js` file? It looks like something that is specific to each form.

Reply:
No that does make sense. Maybe just a short description of what it is for above it so that if it needs to be changed if backend constants from the server changes then it will be easy for Steven to understand what it does.

Reply:
Updated to provide a description.